Automattic/wp-e2e-tests
# Adds Gutenberg Editor Specs

This is a part of #1620 

This adds a new test site `https://e2eflowtestinggutenbergsimple.wordpress.com` which is opted in for Gutenberg, and this copies the editor specs to use this new site. 

**We need to keep the existing editor tests for now until 100% of users are using Gutenberg and the Calypso editor is removed completely.**

**Testing Instructions**

1. You'll need to decrypt the config for the new account
2. Make sure the new editor spec `Basic Public Post` works for the new Gutenberg simple site
3. Make sure none of the non-converted specs run

@Automattic/flowpatrol feel free to merge this if if makes sense to continue work on #1620 and since config is binary we should avoid merge conflicts
